packages/msgpack/msgpack.1.2.1/opam

```diff
@@ -14,7 +17,10 @@ install: [
 remove: [
   ["ocamlfind" "remove" "msgpack"]
 ]
-depends: ["ocamlfind" "camlp4"]
+depends: [
+  "ocamlfind"
```

Comment:
I don't think that's a problem: `{build}` just means "don't recompile the package when the dependency is updated".